packages/manager/modules/pci/src/projects/project/sidebar/sidebar.constant.js

```diff
@@ -143,6 +143,7 @@ export const getMenu = ({ DBAAS_LOGS_URL }) => [
         },
         title: 'Notebooks',
         regions: ['CA', 'EU'],
+        feature: 'notebooks',
```

Comment:
👍  
you can remove the region. Its not required when feature is set.

Reply:
This is done. Thanks :)